### DISCUSSION TITLE
WebM support

### DISCUSSION THREAD

**Discussion by pasaranax** [continue-revolution/sd-webui-animatediff#296] [Ideas]
Hi! Are you going to support output in webm? Webp is not supported on civitai (only the first frame is visible), mp4 and gif don't support metadata.

**Comment by continue-revolution** [comment]
There is indeed metadata in all formats including mp4 and gif.

@zappityzap can you try to support webm?

**Reply by zappityzap** [reply]
I think I tried once before to add WebM and did not succeed, but I can try again.

MP4 output isn't saving metadata right now. It should be possible to save somewhere in the MP4 file, but A1111 may not be able to read it.

GIF output saves metadata to the comment field, which is visible when loaded in PNG Info tab, but A1111 doesn't see it as "parameters". A1111 seems to look for metadata in PIL Image info field, or EXIF if it exists. piexif claims to support only JPEG and WebP formats.

XMP might be a better format for saving metadata, but I'm not familiar with it yet.

**Comment by continue-revolution** [comment]
#301
